src/ui/components/shared/WorkspaceSettingsModal/utils.tsx

```diff
@@ -55,13 +55,27 @@ export const pricingDetailsForSubscription = (subscription: Subscription): PlanP
         seatPrice: 20,
         trial: isTrial(subscription),
       };
+    case "team-annual-v1":
+      return {
+        billingSchedule: "annual",
+        displayName: "Team",
+        seatPrice: 220,
+        trial: isTrial(subscription),
+      };
     case "org-v1":
       return {
         billingSchedule: "monthly",
         displayName: "Organization",
         seatPrice: 75,
         trial: isTrial(subscription),
       };
+    case "org-annual-v1":
+      return {
+        billingSchedule: "annual",
+        displayName: "Organization",
+        seatPrice: 825,
```

Comment:
We should still display monthly price. We just need a new discount field which we can set to 10%.

Reply:
Right now, we display "cost per seat" and "monthly charge" for monthly renewals. Do you want to show both monthly and annual cost for annual renewals along with the "cost per seat"? What about the discount?

Here's what I have now but I don't think the messaging is clear enough.

![image](https://user-images.githubusercontent.com/788456/150398689-a8489811-71f0-4874-b2ca-5c9156030bbc.png)

Reply:
I think it's better to show

```
Cost  per seat:        $20.00
Discount:                     10%
Savings:                 -$24.00
Annual charge.      $216.00
```

Reply:
Maybe this?

![image](https://user-images.githubusercontent.com/788456/150425725-3545e1a3-4088-4c01-99c1-a7f7efa457fa.png)